packages/fury-adapter-remote/lib/adapter.js

```diff
@@ -0,0 +1,131 @@
+const deckardcain = require('deckardcain');
+const axios = require('axios');
+
+const name = 'remote';
+
+const outputMediaType = 'application/vnd.refract.parse-result+json; version=1.0';
+
+const defaultOptions = {
+
+  // the default value, but consumers should be able to override to use their own deployment
+  url: 'https://api.apielements.org',
+
+  parseEndpoint: '/parser',
+  validateEndpoint: '/validate',
+  serializeEndpoint: '/composer',
+
+  // the collection of "parse", media types we want this
+  // instance of the adapter to handle.
+  // NOTE, this allows you to use the API for one media type but
+  // another local adapter for another.
+
+  mediaTypes: [
+    'text/vnd.apiblueprint',
+    'application/swagger+json',
+    'application/swagger+yaml',
```

Comment:
Please add `application/vnd.oai.openapi` and `application/vnd.oai.openapi+json` to support OAS 3